src/plugin/memory-webview-main.ts

```diff
@@ -124,6 +125,7 @@ export class MemoryWebview {
                     <meta http-equiv='Content-Security-Policy' content="default-src 'none'; font-src ${cspSrc}; style-src ${cspSrc} 'unsafe-inline'; script-src ${cspSrc};">
                     <script type='module' src='${mainUri}'></script>
                     <link href="${codiconsUri}" rel="stylesheet" />
+                    <link href="${memoryInspectorCSS}" rel="stylesheet" />
```

Comment:
Tend to prefer this style with css as an external asset.

Either way I'd like it to be done in a consistent manner (either all extracted or all in code)

Reply:
I was in the middle of refactoring this CSS to CSS in JS but it looks like CSS in JS doesn't support pseudo-class rules (e.g. :hover) without adding some additional utilities (like `styled-components`). Rather than struggling with that, would you mind if we create a separate issue/MR to migrate all existing CSS in JS over to the external stylesheets? Should be a relatively quick task, but would prefer not to clutter this MR